packages.dhall_

```diff
@@ -0,0 +1,5 @@
+let upstream =
```

Comment:
What’s the reason to rename these dhall files? If we’re keeping them around for spago legacy users then we should keep the old names. If we don’t need them anymore we should delete them. cc @colinwahl

Reply:
removed, tnx